### PR TITLE
[API] AdminUser & Customer authentication with API Platform 3

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 2.0                   |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| License         | MIT                                                          |

From now on, we can log in as an Admin and Customer with API Platform 3
image

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Resources/config/integrations/swagger.xml [hunk 1]
```diff
@@ -18,53 +18,53 @@
     <services>
         <defaults public="false" />
 
-        <service id="Sylius\Bundle\ApiBundle\OpenApi\Factory\OpenApiFactory" decorates="api_platform.openapi.factory">
+        <service id="Sylius\Bundle\ApiBundle\OpenApi\Factory\OpenApiFactory" decorates="lexik_jwt_authentication.api_platform.openapi.factory">
```
Comment (Rafikooo) [inline src/Sylius/Bundle/ApiBundle/Resources/config/integrations/swagger.xml:21]: I'm not sure if it is needed

Comment (GSadee) [reply src/Sylius/Bundle/ApiBundle/Resources/config/integrations/swagger.xml:21]: If I see correctly, `lexik_jwt_authentication.api_platform.openapi.factory` decorates `api_platform.openapi.factory`, so this change seems to be ok

**Review by GSadee — APPROVED** [review]
Can you add screenshots from swagger?

**Comment by Rafikooo** [comment]
@GSadee, the screenshot included

**Pull request merged**